app/src/main/res/drawable/bottom_rounded_rect_white_background.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Suggest using "option + cmd + L" in all drawable xml files in this PR.

Reply:
done.